scripts/fetch/fetch_utils.R

```diff
@@ -0,0 +1,62 @@
+# Fetch utility functions
+
+#' Convert a vector of counties into sf polygons
+#' 
+#' counties must be strings of "[state]:[county]". See ?maps::county.fips
+#' 
+county_to_sf <- function(counties, crs = 3086){
+  library(sf)
+  library(maps)
+  
+  sf_poly <- sf::st_as_sf(map("county", region = counties, plot = FALSE, fill=TRUE))
+  sf_poly_transf <- st_transform(sf_poly, crs)
+  
+  return(sf_poly_transf)
+}
+
+#' Convert an sf object into an sp object
+#' 
+sf_to_sp <- function(sf_poly, crs = "+proj=longlat +datum=WGS84 +ellps=WGS84 +towgs84=0,0,0"){
+  library(sf)
+  library(sp)
+  
+  sp_poly <- as(sf_poly, "Spatial")
+  sp_poly_transf <- sp::spTransform(sp_poly, sp::CRS(crs))
+  
+  return(sp_poly_transf)
+}
+
+#' take an sf object polygon and make a grid out of it
+#' 
+#' This really belongs in process, but having fetch depend
+#' on process is not a feature yet.
+#' 
+poly_to_grid <- function(sf_poly, cell_size = 15000, crs = 3086){
```

Comment:
I don't think the crs should be an optional parameter. sf_poly should have a projection and it should just get used throughout.

Reply:
cell_size shouldn't have a default?